topology.yaml

```diff
@@ -0,0 +1,48 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# yaml definition file for the nodebase DSE topology.
+
+name: DSE
+description: A Datastax (DSE) cluster
+
+node groups:
+    kdc-node:
```

Comment:
Is it necessary to define this here? Might just be better to always have it called `kdc`oup` in the `kdc` `NodeGr and not be configurable for simplicity.